charts/openperouter/templates/nodemarker.yaml

```diff
@@ -34,6 +34,13 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ template "openperouter.controller.serviceAccountName" . }}
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/control-plane
+        operator: Exists
```

Comment:
should we have this configurable ? 

i.e. have the nodemarker deployed on the control plane nodes based on some configuration ?